packages/fondue/src/components/RichTextEditor/Plugins/MarkPlugin/CodePlugin/index.ts

```diff
@@ -19,7 +19,13 @@ export class CodePlugin extends Plugin {
     }
 
     plugins(): PlatePlugin[] {
-        return [createCodePlugin()];
+        return [
+            createCodePlugin({
+                options: {
+                    hotkey: [],
```

Comment:
`cmd+shift+c` / `ctrl+shift+c` instead, so we are not just removing the shortcut